server/repository/src/db_diesel/item_link_row.rs

```diff
@@ -64,6 +64,24 @@ impl<'a> ItemLinkRowRepository<'a> {
         Ok(())
     }
 
+    #[cfg(feature = "postgres")]
+    pub fn insert_one_or_ignore(&self, item_link_row: &ItemLinkRow) -> Result<(), RepositoryError> {
+        diesel::insert_into(item_link_dsl::item_link)
+            .values(item_link_row)
+            .on_conflict(item_link::id)
```

Comment:
Be kinda nice if diesel just had an `insert_one_or_ignore` implementation for pgsql that was just like this 🤔... looks like they've gone the other way and settled on the `on_conflict` method for sqlite and pgsql in [2.1.4](https://docs.rs/diesel/2.1.4/diesel/query_builder/struct.InsertStatement.html#method.on_conflict)